lib/neek.js

```diff
@@ -56,14 +56,20 @@ module.exports = {
 
 };
 
-function isReadStream(rs){
-  return isStream(rs, process.stdin, '_read');
+function isReadStream(obj) {
+  // some streams are readable but do not have `read()`, however
+  // we can reliably detect readability by testing for `resume()`
+  return isStream(obj) && typeof obj.resume === 'function';
```

Comment:
The input stream I was using was from a [request](https://github.com/request/request) object (an http file download) and that stream didn't have a read function either.

Here is an example of another project that implemented in this same way:
https://github.com/Kuniwak/chai-stream/blob/master/src/helpers.js

The resume() function is what is actually being used by the pipe() call under the hood so it makes sense to test for that.